.github/workflows/pull_request.yml

```diff
@@ -93,6 +93,9 @@ jobs:
       data: ${{ needs.RunConfig.outputs.data }}
       run_command: |
           python3 fast_test_check.py
+    secrets:
+      secret_envs: |
+        CHCACHE_PASSWORD=${{secrets.CHCACHE_PASSWORD}}
```

Comment:
probably we will need to keep this secret in aws as this one won't be visible for forks

Reply:
Sure, I just had nobody to ask to do it last night :sweat_smile: